test/usecases/func_proc_pnt/simulate.py

```diff
@@ -0,0 +1,15 @@
+from neuron import h
+
+s = h.Section()
+
+pnt_proc = h.test_func_proc_pnt(s(0.5))
+
+pnt_proc.set_x_42()
+
+assert pnt_proc.x == 42
+
+pnt_proc.set_x_a(13.7)
+
+assert pnt_proc.x == 13.7
+
+assert pnt_proc.get_a_42(42) == 84
```

Comment:
just to understand the current convention (/ philosophy): a test like this could also easily cover the purpose of `test/usecases/point_process/simulate.py`. And this could be easily also combined in `test/usecases/global_breakpoint/simulate.py`.

 Do we try to minimise the number of test cases wherever possible? (of course, without making them too much complicate)

Reply:
As Luc suggested in Slack having small and independent tests help with the actual testing and understanding of how NMODL features work. I agree with this for the moment and while the code is actively being developed.